### PR TITLE
Update main.go

### PR DESCRIPTION
**Pull request by sammarth-kapse**
Removed Useless Comments.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -5,7 +5,6 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
-//the main executatble file
```
Comment (sammarth-kapse) [inline main.go]: ghhh

File: main.go [hunk 2]
```diff
@@ -5,7 +5,6 @@ import (
 	"github.com/gin-gonic/gin"
 )
```
Comment (sammarth-kapse) [inline main.go]: hfsehjes

File: main.go [hunk 3]
```diff
@@ -5,7 +5,6 @@ import (
...
 
-//the main executatble file
 //gin router 
 var router *gin.Engine
```
Comment (sammarth-kapse) [inline main.go]: ftyg

Comment (guptaabhijit) [reply main.go]: This is not good commnet

**Pull request merged**